### DISCUSSION TITLE
Github Notion Assignment Import

### DISCUSSION THREAD

**Discussion by Bidso** [JamesNZL/notion-assignment-import#206] [Q&A] [answered]
Trying to update my Canvas to Notion Course Code but keep getting an error with the error "Notion Canvas Assignment Import Authorisation failed. Please try again."  See image below

![Notion Import Error 06-25-23](https://github.com/JamesNZL/notion-assignment-import/assets/137752992/8247978a-ac78-4bc3-a8e5-d17a4487397a)

Notion is an approved integration in Canvas
![Canvas Approved Integrations](https://github.com/JamesNZL/notion-assignment-import/assets/137752992/3ba2205b-2021-4f49-b5ca-b06a71d9a615)

**Comment by JamesNZL** [comment, marked as answer]
Hi!

> Notion is an approved integration in Canvas

Just curious, what led you to think this is necessary?
Nothing done by the extension requires authorisation to Canvas, so you don't need a Canvas API key.

> "Notion Canvas Assignment Import Authorisation failed. Please try again." See image below

How are you trying authorise with Notion?
Are you trying to log in with Google, or are you using your own email/password?
(See the *Warning* under [Step 4 of the README](https://github.com/JamesNZL/notion-assignment-import#setup-instructions))
